bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/aemform/AEMFormImpl.java

```diff
@@ -262,6 +264,11 @@ public boolean isFormSelected() {
         return !FormType.NO_FORM_SELECTED.equals(getFormType());
     }
 
+    @Override
+    public boolean isAdaptiveFormV2() {
```

Comment:
I'd prefer `getVersion` API exposed instead of exposing APIs for every different version in future.

Reply:
Ok, will make this change

Reply:
Hi nitin,
If we expose getVersion then whenever our fd:version is updated all the consuming code of this api needs to make that change but if we expose wrapper methods then that change will be made by us at one place in the model itself.
Isn't that better

Reply:
exposed getVersion Api